spec.go

```diff
@@ -39,8 +37,29 @@ type Interface interface {
 	PullChartTarball(ctx context.Context, tarballURL string) (string, error)
 	// RunReleaseTest runs the tests for a Helm Release. This is the same
 	// action as running the helm test command.
-	RunReleaseTest(ctx context.Context, releaseName string, options ...helm.ReleaseTestOption) error
+	RunReleaseTest(ctx context.Context, releaseName string, options ReleaseTestOptions) error
 	// UpdateReleaseFromTarball updates the given release using the chart packaged
 	// in the tarball.
-	UpdateReleaseFromTarball(ctx context.Context, releaseName, path string, options ...helm.UpdateOption) error
+	UpdateReleaseFromTarball(ctx context.Context, releaseName, chartPath string, values map[string]interface{}, options UpdateOptions) error
+}
+
```

Comment:
These are the options we're using currently. I don't think we need to support all the options.

I think its cleaner to define the options ourselves rather than using the helm types as we did previously.